### ISSUE TITLE
Possible to merge Open in Encoding and Open File together

### ISSUE THREAD

**Issue by Chris2011** [junichi11/netbeans-encoding-plugin#22]
Maybe this is not possible with the plugin but it is an enhancement. In my opinion, there is no need to have a second open file dialog. It would be good to have this merged in the normal open file dialog. If you really want to change the encoding, you can do it, inside the open file dialog. If not, then you don't need to.

If this is not possible, maybe after the move from Oracle to Apache, it could be good, to have this plugin merged into the core, so that you can change the open file dialog logic.

Cheers

Chris

**Comment by junichi11** [comment]
I won't do it, sorry.

**Comment by Chris2011** [comment]
So you say only for the moment right? It could be, that it could be part of NetBeans core after a stable phase? That would make much more sense.

TLDR;
The thing is, you open a file vie Menu -> File, why shouldn't you open it with a specific MimeType by default? This is a good approach for that and will reduce the amount of new actions inside of the menu. I have another plugin Bined (aka DeltaHex) and it has an action called Open As Hex, right inside of the menu -> file.

So in general, it should be a default feature for opening files. There is no need to have another one. Everyone uses only one action like open file and will not notice any other actions which are similar.


Cheers and thanks for the plugin


Chris
